pkg/api/ontapi/zapi/client.go

```diff
@@ -411,3 +421,27 @@ func (c *Client) invoke(withTimers bool) (*node.Node, time.Duration, time.Durati
 
 	return result, responseT, parseT, nil
 }
+
+func (c *Client) TraceLogSet(collectorName string, config *node.Node) {
+	// check for log sets and enable zapi request logging if collectorName is in the set
+	if llogs := config.GetChildS("log"); llogs != nil {
+		for _, log := range llogs.GetAllChildContentS() {
+			if strings.EqualFold(log, collectorName) {
+				c.logZapi = true
+			}
```

Comment:
may be we can stop the loop after EqualFold return true to stop the iterations further.